scripts/Brick.js

```diff
@@ -0,0 +1,30 @@
+const PopUpMessage = require('./PopUpMessage');
+
+
+// The project unit is a color block.
+class Brick{
+  constructor(color) {
+    this.color = color;
+    this.element = document.createElement('div');
+    this.element.className = 'brick';
+    this.element.style.backgroundColor = color;
+    this.element.innerHTML = color.toString();
+    this.element.addEventListener('click', () => {
+      this.element.innerHTML = 'Copied!';
```

Comment:
You should add to class params (in this) just another element, which will be in this brick, but shows topper then brick. Instead of changing innerHTML and class of brick